app/src/org/commcare/views/widgets/MediaWidget.java

```diff
@@ -237,21 +235,12 @@ public void setBinaryData(Object binaryuri) {
             }
         }
 
-        if (newMedia.exists()) {
-            // Add the copy to the content provider
```

Comment:
Though I'm not sure whether we should be adding all the files to the content provider. 
IMO, it makes sense for the video that are recorded using the record button in app, but for videos that are chosen from gallery, we just make a copy of that inside our app's directory so if we add it to content provider there'll be 2 copies of the same video shown to the user in the gallery app.

Reply:
Agree that we should not save it to content Providers. @ctsims we did remove it for Images sometime back in [here](https://github.com/dimagi/commcare-android/pull/2110/files), so guessing no objections in removing them for audio and videos as well.